app/bundles/CampaignBundle/Entity/Campaign.php

```diff
@@ -173,27 +173,24 @@ public static function loadValidatorMetadata (ClassMetadata $metadata)
      */
     public static function loadApiMetadata(ApiMetadataDriver $metadata)
     {
-        $metadata->setGroupPrefix('asset')
+        $metadata->setGroupPrefix('campaign')
             ->addListProperties(
                 array(
                     'id',
-                    'title',
-                    'alias',
+                    'name',
                     'category',
-                    'description'
+                    'description',
+                    'events',
```

Comment:
@escopecz Let's move everything from events  to canvasSettings down into the `addProperties()` method.  `addListProperties()` are more for identifier fields (id, name, description, etc) and the more detailed properties go into `addProperties()`.

Thanks!